lib/telemetry_metrics.ex

```diff
@@ -301,9 +351,21 @@ defmodule Telemetry.Metrics do
 
             # Database Time Metrics
             summary("my_app.repo.query.total_time", unit: {:native, :millisecond}),
-            summary("my_app.repo.query.decode_time", unit: {:native, :millisecond}),
-            summary("my_app.repo.query.query_time", unit: {:native, :millisecond}),
-            summary("my_app.repo.query.queue_time", unit: {:native, :millisecond}),
+            summary(
+              "my_app.repo.query.decode_time",
+              unit: {:native, :millisecond},
+              filter: &(is_number(&1))
+            ),
+            summary(
+              "my_app.repo.query.query_time",
+              unit: {:native, :millisecond},
+              filter: &(is_number(&1))
+            ),
+            summary(
+              "my_app.repo.query.queue_time",
+              unit: {:native, :millisecond},
+              filter: &(is_number(&1))
```

Comment:
Are those examples correct? Shouldn’t it receive all meta? Also, we automatically remove them if they are nil.